modules/ROOT/pages/sdk-xattr-overview.adoc

```diff
@@ -1,15 +1,26 @@
 = Extended Attributes
 
 [abstract]
-Couchbase Server permits the definition of _extended attributes_.
-These allow application developers to define application-specific metadata that will only be visible to those applications that request it or attempt to modify it.
-This might be, for example, meta-data specific to a programming framework that should be hidden by default from other frameworks, or possibly from other versions of the same framework.
+Extended Attributes (XATTR) are metadata that can be provided on a per-application basis.
```

Comment:
hm, this seems to need a "for example".  that might be in the shared:partial, but I don't see it in this commit or in the repo.

Reply:
Have refreshed the tags in the shared-partials file.